--- src/common/read_config.c.orig	2014-05-19 14:42:58.046060363 -0700
+++ src/common/read_config.c	2014-05-19 14:45:58.518056225 -0700
@@ -3611,10 +3611,14 @@
 	if (strcmp(conf->priority_type, "priority/multifactor") == 0) {
 		if ((strcmp(conf->schedtype, "sched/wiki")  == 0) ||
 		    (strcmp(conf->schedtype, "sched/wiki2") == 0)) {
-			error("PriorityType=priority/multifactor is "
-			      "incompatible with SchedulerType=%s",
-			      conf->schedtype);
-			return SLURM_ERROR;
+			if( conf->priority_weight_age || conf->priority_weight_fs ||
+			    conf->priority_weight_js || conf->priority_weight_part ||
+			    conf->priority_weight_qos) {
+				error("PriorityType=priority/multifactor is "
+				      "incompatible with SchedulerType=%s",
+				      conf->schedtype);
+				return SLURM_ERROR;
+			}
 		}
 	}
 	if (conf->preempt_mode) {
